pkg/caretta/ebpf_map.go

```diff
@@ -0,0 +1,37 @@
+package caretta
+
+import "github.com/cilium/ebpf"
+
+type ConnectionsMapIterator interface {
+	Next(interface{}, interface{}) bool
+}
+
+type ConnectionsMap interface {
+	Lookup(interface{}, interface{}) error
+	Iterate() ConnectionsMapIterator
+	Delete(interface{}) error
+}
+
+type EbpfMap struct {
+	innerMap ebpf.Map
```

Comment:
Same here - copying ebpf.Map objects might be an issue, I'd use the pointer directly

Reply:
done